lib/dispatch/dispatch.ex

```diff
@@ -36,7 +36,10 @@ defmodule Dispatch do
   @doc """
   Returns a list of usernames that should be request to review the pull request
   """
-  def fetch_selected_users(repo, stacks, author_username, disable_learners \\ false) do
+  def fetch_selected_users(repo, stacks, author_username, opts \\ []) do
+    disable_learners = Keyword.get(opts, :disable_learners, false)
```

Comment:
Elle sert quand cette variable la?

Reply:
At first, it was the 3rd argument for `fetch_selected_users/3` that I replaced with a keyword list to handle multiple options. The name haven’t changed so that’s why there is not related diff elsewhere.

Reply:
Ah right, I missed that part, damn destructuring hit me on this one.